app/templates/pages/justice-and-other-user.html

```diff
@@ -13,26 +13,28 @@
 {% endblock %}
 
 {% block content %}
-<h1 class="govuk-heading-l">GitHub Organisation Invitation</h1>
+<h1 class="govuk-heading-l">Verify your work email address</h1>
 
 <p class="govuk-body">
   To access the Ministry of Justice GitHub organisations you've selected, your email address <strong>{{email}}</strong>
-  will be verified
-  using Azure Active Directory (AD). This secure step is essential to confirm your identity and initiate the process of
-  joining our GitHub organisations.
+  will be verified using Azure Active Directory (AD).
+  This secure step is essential to confirm your identity and initiate the process of joining our GitHub organisations.
 </p>
 
 <p class="govuk-body">
-  By clicking the button below, you will be redirected to a secure Azure AD login page for authentication.
+  By clicking the button below, you will be redirected to a secure sign-on page where you will be asked to confirm your Office 365 login details.
 </p>
 
 <p class="govuk-body">
-  After successfully authenticating with your @justice.gov.uk credentials, GitHub will send an invitation to
-  <strong>{{email}}</strong>. You will need to accept this invitation to join the selected GitHub organisation.
+  After successfully authenticating with your @justice.gov.uk credentials, GitHub will send an invitation to 
```

Comment:
This section needs to be replaced with "After successful verification a GitHub invitation will be sent to your work email address. You will need to accept this invitation to join the selected GitHub organisation."

Reply:
Line 29 still contains the old text:

> After successfully authenticating with your @justice.gov.uk credentials, GitHub will send an invitation to <strong>{{email}}</strong>. You will need to accept this invitation to join the selected GitHub organisation.

It should just read:

> After successful verification a GitHub invitation will be sent to your work email address. You will need to accept this invitation to join the selected GitHub organisation.